Sources/Meta/Visitors/Visitor.swift

```diff
@@ -0,0 +1,39 @@
+import SwiftSyntax
+
+public class Visitor: SyntaxVisitor {
```

Comment:
`Visitor` is a bit meaningless. I think we need a better name for this. What about `NestedDeclarationVisitor`?

Reply:
Great!